packages/web-client/app/components/card-pay/merchant-payment-request-header/index.hbs

```diff
@@ -1,3 +1,7 @@
+<InHead>
+  {{!-- template-lint-disable no-forbidden-elements --}}
+  <meta property='og:image' content={{this.cardPayLogo}} />
```

Comment:
OG protocol doesn't support SVG, I will change this to a PNG.

Reply:
Also needs to be an absolute URL.

Reply:
Thanks for these fixes!